ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts

```diff
@@ -311,6 +313,14 @@ export class FeedconfigurationComponent implements OnInit, AfterViewInit {
     ngOnInit() {
         this.currentUserWithPerimeters = this.userService.getCurrentUserWithPerimeters();
 
+        this.settingsService.getUserSettings().subscribe({
+            next: (response) => {
+                if (response.status === ServerResponseStatus.OK) {
+                    this.userSettings = response.data;
+                }
+            }
+        });
```

Comment:
The settings are in memory , you can avoid a request on the back : 

this.configService.getConfigValue('settings')

Reply:
Done.